### PR TITLE
Fixes to reported connected sites

### PR DESCRIPTION
**Pull request by grs**

### CODE REVIEW

**Comment by grs** [comment]
This handles the issue described in https://github.com/skupperproject/skupper/issues/145#issuecomment-666575672 by detecting and warning the user if there are multiple edge uplinks to distinct networks.

It also handles the case where there are more edge sites connected and makes similar fixes to interior status as were made already to edge site status.

File: cmd/skupper/skupper.go [hunk 1]
```diff
@@ -416,6 +416,12 @@ func main() {
 				if vir.Status.TransportReadyReplicas == 0 {
 					fmt.Printf("Skupper is enabled for namespace %q%s. Status pending...", cli.Namespace, modedesc)
 				} else {
+					if len(vir.Status.ConnectedSites.Warnings) > 0 {
+						for _, w := range vir.Status.ConnectedSites.Warnings {
+							fmt.Printf("Warning: %s", w)
+							fmt.Println()
```
Comment (nicob87) [inline cmd/skupper/skupper.go:422]: isn't this Println equivalent to add a "\n" at the end of the "Printf"?

File: cmd/skupper/skupper.go [hunk 2]
```diff
@@ -416,6 +416,12 @@ func main() {
 				if vir.Status.TransportReadyReplicas == 0 {
 					fmt.Printf("Skupper is enabled for namespace %q%s. Status pending...", cli.Namespace, modedesc)
 				} else {
+					if len(vir.Status.ConnectedSites.Warnings) > 0 {
```
Comment (nicob87) [inline cmd/skupper/skupper.go:419]: I think this if is not needed, if "len=0" it just do not iterate

Comment (grs) [reply cmd/skupper/skupper.go:424]: The len check will also return 0 if the slice is nil. Arguably you could just check if it is nil or not here, but to me the len is a nice way of checking whether iteration is needed in either case.

Comment (nicob87) [reply cmd/skupper/skupper.go:424]: aha!

File: cmd/skupper/skupper.go (hunk 1 above)
Comment (nicob87) [reply cmd/skupper/skupper.go:427]: no need to change, I was not sure it this was intentional, Thanks!

**Pull request merged**